test/internal/helpers/ports.go

```diff
@@ -0,0 +1,14 @@
+package helpers
+
+import (
+	"testing"
+
+	"github.com/phayes/freeport"
+	"github.com/stretchr/testify/require"
+)
+
+func FreePort(t *testing.T) int {
```

Comment:
Please consider using `GetFreePort` from `test/helpers/ports.go` instead of adding a new one.

Reply:
👍

Reply:
a0db795bf